Documentation/known-issues.md

```diff
@@ -36,6 +36,23 @@ The command you can use is:
 ln -s /usr/local/share/dotnet/dotnet /usr/local/bin
 ```
 
+## On dev builds of the tools, restoring default project from dotnet new fails
+When using non-release versions of the CLI, `dotnet restore` will fail to restore `Microsoft.NETCore.App` because for that particular version it exists on a NuGet feed that is not configured on the machine. This behavior is by design and does not happen with public releases (such as RC2).
+
+**Workaround:** create a `NuGet.config` file in the project directory which contains the following:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+    <clear />
+    <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
```

Comment:
Don't use 'cli-deps'.  Instead use 'dotnet-core'.

'cli-deps' is only for building our branch, and we could break others if they start using it.